lib/processCss.js

```diff
@@ -192,11 +192,11 @@ module.exports = function processCss(inputSource, inputMap, options, callback) {
 	if(minimize) {
 		var cssnano = require("cssnano");
 		var minimizeOptions = assign({}, query.minimize);
-		["zindex", "normalizeUrl", "discardUnused", "mergeIdents", "reduceIdents", "autoprefixer"].forEach(function(name) {
+		["cssDeclarationSorter", "normalizeUrl"].forEach(function(name) {
```

Comment:
cssDeclarationSorter should be safe

Reply:
We can enable it back when https://github.com/cssnano/cssnano/issues/535 was fixed. Anyway, it is safer to start from CSSNano 3 optimization and after some tests, enable more optimizations.